betelgeuse.py

```diff
@@ -248,6 +248,8 @@ def add_test_case(args):
                 test.docstring = test.docstring.decode('utf8')
             test.docstring = RST_PARSER.parse(test.docstring)
 
+        test_desc = test.unexpected_tags.get('description', test.docstring if test.docstring else '')
```

Comment:
Testimony have been update and now the information should be retrieved from the `test.tokens` instead of `test.unexpected_tags`. Also you need to rebase and make sure to add the description to the list of testimony.SETTINGS['tokens'].